DESCRIPTION

```diff
@@ -34,18 +34,15 @@ Imports:
     digest,
     downlit,
     fs (>= 1.3.0),
-    htmltools,
```

Comment:
I removed 3 Imports which puts the number of Imports at 19, removing 2 would have been enough as the limit is 20 at the moment.

Reply:
htmltools is a hard dependency of bslib.